app/grandchallenge/workstation_configs/templates/workstation_configs/workstationconfig_detail.html

```diff
@@ -194,5 +202,13 @@ <h1>Viewer Configuration {{ object.title }}</h1>
             </a>
         </div>
     {% endif %}
+{% endblock %}
 
+{% block script %}
+    {{ block.super }}
+    <script type="text/javascript">
+      $(function() {
+        $('[data-toggle="tooltip"]').tooltip();
+      });
+    </script>
```

Comment:
I would like to add a CSP to grand challenge, so I don't want to add any more inline JS. Please move this to its own file, there might be one that does this already.